gatsby-config.js

```diff
@@ -2,30 +2,31 @@ module.exports = {
   siteMetadata: {
     title: `Wayfair Open Source`,
     description: `Home of the Open Source Program Office and Wayfair specific documenting for working with OSS`,
-    author: ``,
-    siteUrl: `http://wayfair.github.io`,
+    author: `opensource@wayfair.com`,
+    siteUrl: `https://wayfair.github.io`,
+    keywords: [`opensource`, `wayfair`, `Wayfair`, `OSS`],
```

Comment:
does it make sense to also include `OSPO` here?

Reply:
donezo

Reply:
yamzo